src/svgs/icons/index.js

```diff
@@ -9,7 +9,9 @@ export { default as CameraIcon } from './camera-icon.svg';
 export { default as ChatBubbleIcon } from './chat-bubble-icon.svg';
 export { default as CheckmarkIcon } from './checkmark-icon.svg';
 export { default as ChevronIcon } from './chevron-icon.svg';
+export { default as CircleAddIcon } from './circle-add-icon.svg';
 export { default as CircleCheckmarkIcon } from './circle-checkmark-icon.svg';
+export { default as CircleRemoveIcon } from './circle-remove-icon.svg';
```

Comment:
[tiny nit] maybe this should be CircleMinusIcon since its like the minus icon but has a circle around it. Add is more the action and the minus is more of the composition.

Also may apply to CircleAddIcon -> CirclePlusIcon

Reply:
Yeah, that's a good thought. For reference, Semantic UI uses "Plus Circle" and "Minus Circle" to describe similar icons. https://semantic-ui.com/elements/icon.html @Shemoel1 any thoughts?